docs/Guide/application-commands/interaction-handlers/autocomplete.mdx

```diff
@@ -3,6 +3,20 @@ sidebar_position: 2
 title: Autocomplete
 ---
 
-<!-- INFO ON AUTOCOMPLETE INTERACTION HANDLERS AS WELL AS HOW TO USE AUTOCOMPLETERUN -->
+Certain chat input command options (Just the string and number options as of 6/26/2022) can be autocompleted based on what the user has written. There are two ways you can create an autocomplete interaction handler- the way we've done it before with `interaction-handler` methods and through the `autoCompleteRun` method!
 
-<!-- TODO: https://vladfrangu.notion.site/Application-Commands-3d1683f23f0a4fa6b3e682bc8c1cc1b9#ceb1a6f5ec154f85987a95181d512660 -->
+:::warning
+
+You cannot defer auto-completes (that would be silly)! You must respond to the interaction within **3 seconds**.
+
+:::
+
+# Interaction Handler
```

Comment:
Probably should add an example, but I think by this point the reader should already be able to do this.

I think the example would best be for the run method. Any suggestions?